src/screens/products/components/desmos/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useTranslation } from "i18n";
+import { DesmosContentCSS } from "./styles";
+import { Go } from "@icons";
+
+export const DesmosContent = () => {
+  const { t } = useTranslation("products");
+  return (
+    <DesmosContentCSS>
+      <div className="desktopWrapper">
+        <div className="image">
+          <img
+            src="/static/images/assets/desmos-image.png"
+            alt="Forbole Portal Screenshot"
+          ></img>
+        </div>
+        <div className="content">
```

Comment:
`<li>` should be child of `<ul>` and so this should be a `<ul>` but not a `<div>`.
```suggestion
        <ul className="content">
```